demo/react/src/VoiceWidget.tsx

```diff
@@ -60,40 +48,41 @@ export default function VoiceWidget() {
           <input
             type="text"
             name="accessKey"
-            onChange={(value) => setInputValue(value.target.value)}
-            disabled={isLoaded}
+            onChange={(value) => setAccessKey(value.target.value)}
           />
+          <button
+            className="init-button"
+            onClick={() => initEngine()}
+          >
+            Init Porcupine
+          </button>
         </label>
-        <button className="start-button" onClick={() => setAccessKey(inputValue)} disabled={isLoaded}>
-          Start Porcupine
-        </button>
       </h3>
-      <h3>Dynamic Import Loaded: {JSON.stringify(isChunkLoaded)}</h3>
-      <h3>Porcupine Loaded: {JSON.stringify(isLoaded)}</h3>
+      <h3>Loaded: {JSON.stringify(isLoaded)}</h3>
       <h3>Listening: {JSON.stringify(isListening)}</h3>
-      <h3>Error: {JSON.stringify(isError)}</h3>
-      {isError && accessKey && (
-        <p className="error-message">{JSON.stringify(errorMessage)}</p>
+      <h3>Error: {JSON.stringify(error !== null)}</h3>
+      {error && (
+        <p className="error-message">{JSON.stringify(error)}</p>
       )}
       <h3>Keywords: {JSON.stringify(keywords)}</h3>
-      <br />
+      <br/>
       <button
         onClick={() => start()}
-        disabled={isError || isListening || !isLoaded}
+        disabled={error !== null || !isLoaded || isListening}
       >
         Start
       </button>
       <button
-        onClick={() => pause()}
-        disabled={isError || !isListening || !isLoaded}
+        onClick={() => stop()}
+        disabled={error !== null ||!isLoaded || !isListening}
       >
-        Pause
+        Stop
       </button>
       <button
-        onClick={() => stop()}
-        disabled={isError || !isListening || !isLoaded}
+        onClick={() => release()}
+        disabled={error !== null || !isLoaded}
       >
-        Stop
+        Release
```

Comment:
not sure we need a `Release` button. It's probably better to show them how to do it in a component unmount